job('deliberate-fail'){
    throw new Exception("Arrrgggghhh")
}